dev-util/distro-info/distro-info-1.0.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+DISTUTILS_OPTIONAL=true
+
+inherit distutils-r1
+
+DESCRIPTION="Provides information about the Debian distributions' releases"
+HOMEPAGE="https://debian.org"
+SRC_URI="mirror://debian/pool/main/d/${PN}/${PN}_${PV}.tar.xz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="python test"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+COMMON_DEPEND="dev-lang/perl:=
+	python? ( ${PYTHON_DEPS} )"
+DEPEND="${COMMON_DEPEND}
+	python? ( dev-python/setuptools[${PYTHON_USEDEP}] )
+	test? (
+		dev-util/shunit2
+		dev-python/pylint[${PYTHON_USEDEP}]
```

Comment:
Can we drop the pylint dep? It's not really useful for users

Reply:
I can't get your idea. How we can (and why we should) drop test-time dependency?

Reply:
If it's possible, dropping pylint from the test suite is helpful. We don't really care if formatting standards aren't met -- that's an upstream problem and doesn't indicate if the application works as expected.

pylint is an unnecessary dependency for our case - testing if the application works properly. It's similar to [this](https://dev.gentoo.org/~mgorny/python-guide/pytest.html?highlight=tests#avoiding-dependencies-on-other-pytest-plugins) where coverage tests aren't helpful for end-users.

It's not a big deal, just a suggestion.

Reply:
@thesamesam could you please open a bug for it? I'm all for removing useless deps if it's possible.